resources/features/PAYE.feature

```diff
@@ -0,0 +1,196 @@
+Feature: PAYE Calculation
+    The PAYE Calculation method will take information from the BI data and PAYE refs tables to calculate paye_employees and paye_jobs
+
+    paye_employees takes sum of the average number of jobs per quarter across an id for each ern, so for each id sum the jobs for all the quaters and divide by the number of non-null quaters then sum for each ern
```

Comment:
Can we make it clear that it is not a 'sum all quarters' approach but instead we sum by quarter first to identify the number of non-null quarters, then sum across the quarters and divide etc.